index.html

```diff
@@ -0,0 +1,132 @@
+<!DOCTYPE html>
+<html>
+
+<head>
+  <meta charset="utf-8" />
+  <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+  <title>Graffiti bootcamp - project 3</title>
+  <!-- dont forget to add a css file and link it here! -->
+  <link rel="stylesheet" href="style.css">
+  <!-- Google fonts-->
+  <link href="https://fonts.googleapis.com/css?family=Open+Sans|Raleway:300,400" rel="stylesheet">
+  <link rel="stylesheet" href="https://www.w3schools.com/w3css/4/w3.css">
+</head>
+
+<body>
+  <header>
+    <!-- video or image as a header is cool :) -->
+    <div id=hero>
```

Comment:
When I saw this, I did a quick search for "#hero" in your code, and only found CSS usages.  Consider a class.
Also, include quotations
`<div id="hero">`